src/main/java/com/google/api/generator/engine/writer/JavaWriterVisitor.java

```diff
@@ -127,6 +128,15 @@ public void visit(VariableExpr variableExpr) {
     variable.identifier().accept(this);
   }
 
+  @Override
+  public void visit(TernaryExpr ternaryExpr){
+    ternaryExpr.conditionExpr().accept(this);
+    buffer.append(" ? ");
```

Comment:
Could we please use string tokens as in PR #67?

Reply:
Sounds cool!